Project.toml

```diff
@@ -11,7 +13,7 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 [compat]
 LinQuadOptInterface = "~0.6.0"
 MathProgBase = "~0.5.0, ~0.6, ~0.7"
-julia = "≥ 1"
+julia = "~1"
```

Comment:
I think "1" is sufficient to include all 1.x releases.

Reply:
I haven't quite got my head around this new format.